robot-db/src/main/java/domain/Book.java

```diff
@@ -0,0 +1,75 @@
+package domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+
+/**
+ * Book representation in database.
+ */
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Book {
+
+    @Id
+    @GeneratedValue
+    @Setter
+    @Getter
+    private long bookID;
+
+    @Getter
+    @Setter
+    @Column
+    private String title;
+
+    @Getter
+    @Setter
+    @Column(name = "new_price")
+    private float newPrice;
+
+    @Getter
+    @Setter
+    @Column(name = "old_price")
+    private float oldPrice;
+
+
+    @Getter
+    @Setter
+    @Column(length = 3)
+    private String currency;
```

Comment:
I'd note which currency standard we're following. ISO-number goes here or some other code?